content/en/ecosystem/registry/_index.md

```diff
@@ -11,19 +25,13 @@ description: >-
 # redirect rule to avoid the loop, as suggested by Netlify support
 # (email support ID 159489):
 redirects: [{ from: /ecosystem/registry*, to: '/ecosystem/registry?' }]
-aliases: [/registry/*]
-type: default
-layout: registry
-outputs: [html, json]
-body_class: registry td-content
-weight: 20
 ---
 
 {{% blocks/lead color="dark" %}}
 
 <!-- markdownlint-disable single-h1 -->
 
-# {{% param title %}}
+<h1>{{% param title %}}</h1>
```

Comment:
Avoid the render-heading hook, otherwise the heading is rendered with an anchor self-link.